app/models/step.rb

```diff
@@ -33,6 +34,10 @@ def pre_order_tree_traversal
     [self] + self.child_approvals.flat_map(&:pre_order_tree_traversal)
   end
 
+  def completed_by
```

Comment:
oh wait, there is one, just didn't see it because it is not wrapped in a `describe "#completed_by"` block -- can we add that ?